app/api/admin_communications.py

```
@@ -0,0 +1,450 @@
+        # TODO: Add subscription expiry logic
+        # This would require a subscription table with expiry dates
+        
+    elif request.reminder_type == "credits_low":
+        # Find users with low credit balances
+        threshold = request.threshold or 10
+        
+        result = await db.execute(
+            select(User).where(
+                and_(
+                    User.credits_balance < threshold,
+                    User.is_active == True
```

Ruff (0.13.2)

Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)